tests/providers/apache/drill/operators/test_drill.py

```diff
@@ -56,8 +56,8 @@ def test_drill_operator_single(self):
 
     def test_drill_operator_multi(self):
         sql = [
-            "create table dfs.tmp.test_airflow as" "select * from cp.`employee.json` limit 10",
-            "select sum(employee_id), any_value(full_name)" "from dfs.tmp.test_airflow",
+            "create table dfs.tmp.test_airflow as select * from cp.`employee.json` limit 10",
+            "select sum(employee_id), any_value(full_name) from dfs.tmp.test_airflow",
```

Comment:
I'm not sure why this worked before, but this should be right 🤷‍♂️

Reply:
Becuase this is valid Python 🙂 In terms of Black this is a known (and relatively long-standing) bug.

Reply:
It probably does not, this entire test class is probably not run on the CI (due to the `backend` marker).